controller/homeRoutes.js

```diff
@@ -0,0 +1,82 @@
+const router = require('express').Router();
+const { User, Restaurant, Review } = require('../models');
+const withAuth = require('../utils/auth');
+
+router.get('/', async (req, res) => {
+  try {
+    // Get all Restaurants and JOIN with user data
+    const restaurantData = await Restaurant.findAll({
+    //   include: [
+    //     {
+    //       model: User,
+    //       attributes: ['name'],
+    //     },
+    //   ],
+    });
```

Comment:
I had to comment the include contents out since the User doesn't appear to have any association with the Restaurant model. We might need to add something back in later.